server/app/controllers/api/v1/auth_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Api::V1::AuthController < ApiController
+  def register
+    result = AuthenticationService.register(
+      email: user_params[:email],
```

Comment:
I am fine doing the "foo: bar" instead of ":food => bar" but we should be standard across this codebase. Just a callout.

Reply:
Gotta get that Rubocop

Reply:
Good point. Let me add that in a follow up and i'll fix all the rubocop errors while I'm at it

Reply:
Added here https://github.com/asleepysheepy/industry_tip_jars/pull/13